identity_account/src/types/identity_state.rs

```diff
@@ -0,0 +1,79 @@
+// Copyright 2020-2022 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use identity_account_storage::identity::ChainState;
+use identity_iota_core::document::IotaDocument;
+use serde::Deserialize;
+use serde::Serialize;
+
+use crate::error::Error;
+use crate::error::Result;
+
+/// Holds the internal state for the identity.
+#[derive(Clone, Debug, PartialEq, Deserialize, Serialize)]
+pub struct IdentityState {
```

Comment:
This does not have to be public, I think. Since we store arbitrary blobs we don't need to expose the exact storage layout.